sources/packages/backend/apps/db-migrations/src/sql/Types/Create-cas-invoice-status.sql

```diff
@@ -0,0 +1 @@
+CREATE TYPE sims.cas_invoice_status AS ENUM ('Pending', 'Sent', 'Manual intervention');
```

Comment:
Just a minor thought. Will it be more useful, if we create in the order of `'Pending', 'Manual intervention',  'Sent'` if we are trying to sort invoices by invoice status without using case just following natural enum order. ?

Reply:
There is no clear sorting requirement right now.